mlir/include/mlir/Dialect/Transform/IR/TransformInterfaces.td

```diff
@@ -284,8 +284,8 @@ def TypeConverterBuilderOpInterface
     : OpInterface<"TypeConverterBuilderOpInterface"> {
   let description = [{
     This interface should be implemented by ops that specify a type converter
-    for a dialect conversion. Such ops can be used with
-    "apply_conversion_patterns".
+    for a dialect conversion, or to populate a type converter with
+    conversions. Such ops can be used with "apply_conversion_patterns".
```

Comment:
Nit: please provide some explanation as to when to implement each of the methods.